### PR TITLE
Add RandomElementsFromArray function to random helpers

### PR DESCRIPTION
**Pull request by ryan-gang**
This pull request adds a new function, RandomElementsFromArray, to the random helpers module. This function allows for the random selection of a specified number of unique elements from an array. It also includes a helper function, contains, to check if a value is already present in an array. This addition enhances the functionality of the random helpers module and improves code reusability.

### CODE REVIEW

File: random/helpers.go [hunk 1]
```diff
@@ -68,6 +68,30 @@ func RandomStrings(n int) []string {
 	return l
 }
 
+func RandomElementFromArray(arr []string) string {
```
Comment (rohitpaulk) [inline random/helpers.go:71]: Can we make this generic too?

**Review by rohitpaulk — APPROVED** [review]

File: random/helpers.go [hunk 2]
```diff
@@ -68,6 +68,30 @@ func RandomStrings(n int) []string {
...
+	return RandomElementsFromArray(arr, 1)[0]
+}
+
+func RandomElementsFromArray[T any](arr []T, count int) []T {
```
Comment (rohitpaulk) [inline random/helpers.go:75]: This and the `shuffle` method below are kinda similar btw - wonder if we can remove `shuffle` then?

File: random/helpers.go [hunk 3]
```diff
@@ -68,6 +68,30 @@ func RandomStrings(n int) []string {
...
+	// Randomly selects `count` unique elements from the given array
+	// and returns them in a new array.
+	var seen []int
+	for count > len(arr) {
+		// If we need more elements than the array has, we'll just append the array to itself repeatedly.
+		arr = append(arr, arr...)
+	}
+	elements := make([]T, count)
+	for i := 0; i < count; i++ {
+		index := rand.Intn(len(arr))
```
Comment (rohitpaulk) [inline random/helpers.go:85]: This can be simplified - look at what `shuffle` does with `rand.Perm`

File: random/helpers.go (hunk 2 above)
Comment (ryan-gang) [reply random/helpers.go:75]: Yes, makes sense. 
Have removed it.

**Pull request merged**